mcsema/BC/Function.cpp

```diff
@@ -260,6 +288,132 @@ static llvm::BasicBlock *GetOrCreateBlock(TranslationContext &ctx,
   return block;
 }
 
+static void LiftExceptionFrameLP(TranslationContext &ctx,
+                                 const NativeFunction *cfg_func) {
+
+  if (cfg_func->eh_frame.size() > 0) {
+
+    // The personality function `__gxx_personality_v0` is lifted as global variable. Erase the
+    // variable before declaring it as the function.
+    if (auto llvm_used = gModule->getGlobalVariable("__gxx_personality_v0")) {
```

Comment:
Is this an artifact of current CFG recovery ( sounds like yes? )

Reply:
Also, maybe change the variable name `llvm_used` to something like `personality_func_var`.

Reply:
Yes, the CFG recovers `__gxx_personality_v0` as global variable which conflicts with the personality function declaration. I will change the name to `personality_func_var`